src/System.Collections/src/System/Collections/BitArray.cs

```diff
@@ -597,7 +730,21 @@ public int Length
             }
         }
 
-        public void CopyTo(Array array, int index)
+        // The mask used when shuffling a single int into Vector128/256.
+        // On little endian machines, the lower 8 bits of int belong in the first byte, next lower 8 in the second and so on.
+        // We place the bytes that contain the bits to its respective byte so that we can mask out only the relevant bits later.
+        private static ReadOnlySpan<byte> ShuffleMask_CopyToBoolArray => new byte[] { 0, 0, 0, 0, 0, 0, 0, 0,
```

Comment:
it can be made a `static readonly Vector<byte>` field

Reply:
I was thinking about the [Roslyn's optimisation](https://github.com/dotnet/roslyn/pull/24621) where this kind of pattern will be translated to a span pointing to the compile time constant data. Is the JIT able to do similar things for vector fields?

Reply:
even if it can - a Vector is still better because you don't have to write `fixed + load` in the code.